src/components/TournamentList/TournamentList.js

```diff
@@ -1,6 +1,8 @@
 import React, {Component} from 'react';
 import Tournament from '../Tournament/Tournament';
 
+import './tournament-list.scss';
+
```

Comment:
Please ```use import style from '...'```
then ```className={style['tournament-count]'} ```or with dot notation,

Reply:
Please, provide me with more info about advantages of this approach.

Reply:
Pros:
1. You will get error if you use  a class which doesn't exist in css/scss file
2. Style object can be passed to children
3. Using themr package
4. Each class have unique name (loader adds hash to each class's name, so you can have two classes with identical names and you will be sure that each style applied properly)
5. Optimization - removing unused classes from final build (i'm  not sure if it works by default)


Cons:
1. Using camelCase for naming classes (as I remember  we can specify ```camelCase: true``` and continue to write code in usual way)
2. You have to use styles object whenever constructing a className

Reply:
We use component model, so every component has it's own stylesheet file, anyway each class have unique name (1,4), anyway we don't have unused classes (5) and in this case we don't need to pass style object to children. It will be great approach in any other cases, but now i don't think so. Because at least it makes code ugly...

Reply:
ok, I don't mind. I fully agree with you:)